### PR TITLE
[PowerMock] LPS 151818

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @tinatian @dantewang,

This pr is to remove powermock in `dynamic-data-mapping-service`.

https://issues.liferay.com/browse/LPS-151818

Thanks for checking.

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxMultipleDDMFormFieldTypeReportProcessorTest.java [hunk 1]
```diff
@@ -51,18 +46,19 @@
...
 		).thenReturn(
 			"field1"
 		);
 
-		when(
+		Mockito.when(
 			ddmFormFieldValue.getType()
 		).thenReturn(
-			DDMFormFieldType.CHECKBOX_MULTIPLE
+			DDMFormFieldTypeConstants.CHECKBOX_MULTIPLE
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxMultipleDDMFormFieldTypeReportProcessorTest.java:61]: @JanisZhang Here the deprecated class usages are replaced. Could you make these changes in a separate SF commit? Basically keep the test changes all about tests.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxMultipleDDMFormFieldTypeReportProcessorTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxMultipleDDMFormFieldTypeReportProcessorTest.java:61]: @dantewang sure.

**Comment by tinatian** [comment]
ci:test:relevant

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/116468

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**